contracts/src/utils/SparseBitmap.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.25;
+
+struct SparseBitmap {
```

Comment:
What's the benifit comparing with https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/structs/BitMaps.sol?

Reply:
Ah nice find. Will try to incorporate it if its better. Though I don't want to start relying too much on OpenZeppelin.